### PR TITLE
feat: add Harmony Testnet

### PR DESCRIPTION
**Pull request by JaunePomme**
### Description

Add Harmony testnet
Found the information on the Harmony explorer, the docs and chainList.
### Backward compatibility

No
### Testing

Yes

### CODE REVIEW

File: .changeset/unlucky-windows-compete.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
```
Comment (jmrossy) [inline .changeset/unlucky-windows-compete.md:2]:
```suggestion
'@hyperlane-xyz/registry': minor
```

File: chains/harmonytestnet/logo.svg
Comment (jmrossy) [inline chains/harmonytestnet/logo.svg:1]: This is not a valid SVG image. That's why github isn't showing the preview here

File: .changeset/unlucky-windows-compete.md (hunk 1 above)
Comment (JaunePomme) [reply .changeset/unlucky-windows-compete.md:2]: Yes, fixed.

File: chains/harmonytestnet/logo.svg
Comment (JaunePomme) [reply chains/harmonytestnet/logo.svg:1]: I changed it, should be better now, thx

Comment (JaunePomme) [reply chains/harmonytestnet/logo.svg:1]: Found the right one, fixed

File: chains/harmonytestnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.s0.b.hmny.io
+    family: Harmony
```
Comment (jmrossy) [inline chains/harmonytestnet/metadata.yaml:4]: This must be `etherscan` or `blockscout` or `other

**Review by jmrossy — COMMENTED** [review]
Almost done. There's a problem with the explorer definition and you need to run `yarn prettier` to fix a minor formatting issue with your file

File: .changeset/unlucky-windows-compete.md (hunk 1 above)
Comment (jmrossy) [reply .changeset/unlucky-windows-compete.md:2]: @JaunePomme I've noticed you fix these by adding new files and creating duplicate changesets. I've fixed it for you a few times but I'd appreciate it if you could just modify the existing file. All you have to do is click the `commit suggest` button on my first comment

Comment (JaunePomme) [reply .changeset/unlucky-windows-compete.md:2]: My bad, I'm used to Gitlab and not Github, and never used the changeset system so I kept creating new ones;
Thank you, I'll keep learning and do my best !

File: .changeset/unlucky-windows-compete.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': minor
+---
+
+add Harmony testnet
```
Comment (xeno097) [inline .changeset/unlucky-windows-compete.md:5]: I think that you can remove one of the changeset files as it is duplicated

**Review by xeno097 — CHANGES_REQUESTED** [review]

File: chains/harmonytestnet/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  name: JaunePomme
+  url: https://github.com/JaunePomme
+displayName: Harmony Testnet
+domainId: 1666700000
+isTestnet: true
+name: harmonytestnet
+nativeToken:
+  decimals: 18
+  name: Ether
+  symbol: ETH
```
[the remark this answers is not in the document]
Comment (JaunePomme) [reply chains/harmonytestnet/metadata.yaml]: Indeed, thank you

File: chains/harmonytestnet/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.s0.b.hmny.io
```
[the remark this answers is not in the document]
Comment (JaunePomme) [reply chains/harmonytestnet/metadata.yaml]: Huh, took me a while to find the API and you found a correct one, thanks !

File: .changeset/unlucky-windows-compete.md (hunk 3 above)
Comment (JaunePomme) [reply .changeset/unlucky-windows-compete.md]: I deleted one, I hope it will be enough ; thanks

**Comment by JaunePomme** [comment]
I hope I fixed everything now

**Review by xeno097 — APPROVED** [review]

**Pull request merged**